### ISSUE TITLE
mpcinfo plugin doesn't work

### ISSUE THREAD

**Issue by JohnFlowerful** [hexchat/hexchat#1682]
Noticed that mpc returns 'pattern not found' when I switched to win10 from win8.1. I tested older versions of MPC-HC, but none of them worked.
Then I thought that maybe it's win10, so I tested multiple HexChat and MPC-HC variants in a win7 SP1 VM. No dice.
I didn't test win8.1.

The MPC-HC window title doesn't contain the ' - Media Player Classic' suffix. Perhaps this is the issue?

**Comment by hxe** [comment]
yeah i never had any luck with it either. there is this script tho https://gist.github.com/berrythesoftwarecodeprogrammar/b02ca24bc7834519c40e51ecc9b39131
just make sure you have mpc's web interface enabled. `View->Options...->Player->Web Interface` and make sure the script is pointing to the right host and port

**Comment by TingPing** [comment]
Probably just requires configuration, its an overly complex and awful plugin. Maybe I will finally just remove it this cycle.

**Comment by JohnFlowerful** [comment]
@hxe it used to work for me in win8.1 and continues to work for a guy after upgrading to win10 from win7. It's a weird issue.

**Comment by VADemon** [comment]
Windows 7 SP1,
#1, MPC: 1.7.5.107: Window title is simply the <Filename.ext> of the played file
#2, MPC: 1.7.10: Same.

The plugin prints "pattern not found", basically this [check](https://github.com/hexchat/hexchat/blob/master/plugins/mpcinfo/mpcInfo.c#L53) always fails.
